File: Data Structures & Algorithms/missing-number/submission-4.java
class Solution {
    public int missingNumber(int[] nums) {
        int total=0;
        int res;
        int len = nums.length;
        for(int i =0;i<=len;i++){
            total = total + i;
        }
        for(int i =0;i<nums.length;i++){
            total = total - nums[i];
        }
        return total;
    }
}

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-4.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        for i in range(1,len(nums)):
            if nums[i]<nums[i-1]:
                return nums[i]
        if nums[0]>nums[-1]:
            return nums[-1]
        return nums[0]